pokemongo_bot/cell_workers/evolve_all.py

```diff
@@ -9,9 +9,10 @@ def initialize(self):
         self.evolve_speed = self.config.get('evolve_speed', 3.7)
         self.evolve_cp_min = self.config.get('evolve_cp_min', 300)
         self.use_lucky_egg = self.config.get('use_lucky_egg', False)
+        self._validate_config()
 
     def _validate_config(self):
-        if isinstance(self.evolve_all, str):
+        if isinstance(self.evolve_all, str) or isinstance(self.evolve_all, unicode):
```

Comment:
Check if both are an instance of `basestring` instead. `str` and `unicode` inherits from `basestring`.